Use_Cases/Related_Lists/README.md

```diff
@@ -0,0 +1,26 @@
+# Better Related Lists
+
+The Lightning Account detail page is extended by using Skuid to display related Opportunities and Cases with advanced filtering, sorting, and interactive capabilities. The related lists have multi-field sorting, 10+ columns that can be shown/hidden by individual end users, dynamic filtering of opportunities, two views of Cases (one tab displays open cases, second tab displays closed cases), and a dashboard displaying real time opportunity information.
```

Comment:
It's preferable to capitalize Opportunity and Case when talking specifically about the Salesforce objects, but in general usage (i.e. discussing case and opportunity records) the words should be lower case.

```suggestion
The Lightning Account detail page is extended by using Skuid to display related opportunities and cases with advanced filtering, sorting, and interactive capabilities. The related lists have multi-field sorting, 10+ columns that can be shown/hidden by individual end users, dynamic filtering of opportunities, two views of cases (one tab displays open cases, second tab displays closed cases), and a dashboard displaying real time opportunity information.
```